src/view/widget/tags.jsx

```diff
@@ -22,7 +22,7 @@ const { cacheComponent } = require('../../util/cache');
  */
 class Tags extends Component {
   render() {
-    const { tags, title, showCount } = this.props;
+    const { tags, title, show_count } = this.props;
```

Comment:
Don't need to change `showCount` to `show_count` here.

Reply:
OK, now I changed it back.